src/nvim/channel.c

```diff
@@ -302,7 +305,7 @@ static void close_cb(Stream *stream, void *data)
 Channel *channel_job_start(char **argv, CallbackReader on_stdout,
                            CallbackReader on_stderr, Callback on_exit,
                            bool pty, bool rpc, bool overlapped, bool detach,
-                           const char *cwd,
+                           ChannelStdinMode stdin, const char *cwd,
```

Comment:
"stdin" is a macro in microsoft compilers so we need to change it, maybe `stdin_mode`.

Reply:
Thanks, fixed in f43d493